lib/matplotlib/tests/test_image.py

```diff
@@ -220,12 +220,8 @@ def test_imshow_zoom(fig_test, fig_ref):
 def test_imshow_pil(fig_test, fig_ref):
     style.use("default")
     PIL = pytest.importorskip("PIL")
```

Comment:
you can also get rid of this importorskip (which occurs in a few places)

Reply:
done.